pom.xml

```diff
@@ -1693,6 +1693,21 @@
         <github.nick>mliradelc</github.nick>
       </properties>
     </developer>
+    <developer>
+      <id>awilken</id>
+      <name>Arne Wilken</name>
+      <email>wilken@elan-ev.de</email>
+      <organization>ELAN e.V.</organization>
+      <organizationUrl>https://elan-ev.de/</organizationUrl>
+      <roles>
+        <role>developer</role>
+      </roles>
+      <timezone>Europe/Berlin</timezone>
+      <properties>
+        <irc.nick>Arne Wilken</irc.nick>
```

Comment:
I don't think this is your IRC nick, is it?
If I remember correctly, Freenode (and maybe even IRC) does not allow spaces.

Reply:
...Come to think of it, should I rather be putting my Matrix id there (e.g. `@arnewilken:chat.virtuos.uni-osnabrueck.de`)? Seeing as I don't really use IRC, actually.

Reply:
Doesn't make sense as `<irc.nick>` specifically  mentions IRC.
I don't care if you want to add something like `<matrix.user>` as this isn't used anywhere anyway.